crates/common/light_client/src/header.rs

```diff
@@ -0,0 +1,7 @@
+use ream_consensus::beacon_block_header::BeaconBlockHeader;
+use serde::Serialize;
+
+#[derive(Serialize)]
+pub struct LightClientHeader {
+    pub beacon: BeaconBlockHeader,
+}
```

Comment:
This implementation is outdated https://ethereum.github.io/consensus-specs/specs/capella/light-client/sync-protocol/#modified-lightclientheader